webui/react/src/components/TaskCard.tsx

```diff
@@ -6,14 +6,16 @@ import Icon from 'components/Icon';
 import Link from 'components/Link';
 import ProgressBar from 'components/ProgressBar';
 import TaskActionDropdown from 'components/TaskActionDropdown';
-import { RecentCommandTask, RecentTask } from 'types';
+import { AnyTask, DetailedUser, RecentCommandTask, RecentEvent } from 'types';
 import { percent } from 'utils/number';
 import { canBeOpened, isExperimentTask } from 'utils/task';
 import { openCommand } from 'wait';
 
 import css from './TaskCard.module.scss';
 
-const TaskCard: React.FC<RecentTask> = (props: RecentTask) => {
+type Props = AnyTask & RecentEvent & {curUser?: DetailedUser}
+
+const TaskCard: React.FC<Props> = (props: Props) => {
```

Comment:
nit: destructure props to maintain consistency?

Reply:
do you mean to separate curUser out of props so it is a proper Task or to destructure everything

Reply:
https://github.com/determined-ai/determined/pull/2775/commits/39667be37ec27ad445ad3587fefa4ede006e12b6

Reply:
I meant destructure everything, but this is fine imo.